Makefile

```diff
@@ -1,7 +1,11 @@
+ifdef GREP
+	GREPARG = -g $(GREP)
+endif
+
 REPORTER ?= spec
-UNIT_TESTS = $(shell find ./test/unit -name "test_*.js")
-QPID_INTEGRATION_TESTS = $(shell find ./test/integration/qpid -name "*.test.js")
-SERVICEBUS_INTEGRATION_TESTS = $(shell find ./test/integration/servicebus -name "*.test.js")
+UNIT_TESTS = ./test/unit
+QPID_INTEGRATION_TESTS = ./test/integration/qpid
```

Comment:
I'm not sure how mine is specifying explicit paths - it's using the same paths that yours had, I'm just letting Mocha do the hard work of finding the tests instead of us doing it.